R/test-elements.R

```diff
@@ -1091,16 +1103,44 @@ checkbox_page <-
     )
     get_answer <- function(input, ...)
       if (is.null(input[[label]])) {
-        character(0)
+        ""
       } else {
         input[[label]]
       }
-    validate <- function(answer, ...)
-      if (length(answer) == 0L && force_answer) {
-        failed_validation_message
-      } else {
-        TRUE
+    validate <- function(answer, ...){
+      ret <- TRUE
+      if(force_answer == "one"){
+        if (!any(nzchar(answer))){
+          if("one" %in% names(failed_validation_message)){
+            ret <- failed_validation_message[["one"]]
+          }
+          else{
+            ret <- failed_validation_message[[1]]
+          }
+          }
+        }
+      if(force_answer == "first"){
```

Comment:
For clarity maybe write `else if` instead of just `if`?

Reply:
Done